Updating Dev with UI updates to Module Creation

dmols opened the pull request:
We have ModuleCreator and ModuleSkeleton. ModuleCreator is the base, and ModuleSkeleton is what could become of ModuleCreator once we fully develop a working system to store a new module and decide to up the complexity of a module.

Michael-Dews committed:
Initial commit, not ready to merge yet

mbrancato31 committed:
Merge pull request #69 from nlyra/dev

getting latest from dev

Commit:
Merge remote-tracking branch 'origin/39-ModuleCreation-Cards' into 39-ModuleCreation-Skeleton

Commit:
Updated some paths to styles pages

Commit:
Merge branch 'ModuleCreation-Skeleton' of https://github.com/nlyra/SystemsEngineeringToolkit into 39-ModuleCreation-Skeleton

Commit:
Updating App.js because of duplicate imports

mbrancato31 reviewed (commented):
i guess is fine, but should choose one type od module creator, this request has 2, it may make some issues in the long run with dev

mbrancato31 committed:
Front end connected to backend

- saves basic information of the module into the array in the course module
- changed body variable to description variable to keep consistency

mbrancato31 committed:
Merge branch 'dev' into ModuleCreation-Skeleton

mbrancato31 committed:
small fixes

dmols committed:
Updated Login page routing

dmols reviewed (commented):
Merging into dev, after reviewing changes made to the files.
